src/read.c

```diff
@@ -4992,6 +5148,94 @@ avifResult avifDecoderReset(avifDecoder * decoder)
         }
 #endif // AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP
 
+#if defined(AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM)
+        // AVIF_ITEM_SAMPLE_TRANSFORM (not used through mainItems because not a coded item (well grids neither but it's different)).
```

Comment:
Nit: "well grids neither" is not clear. Does it mean "grids are not used through mainItems either" or "grids are not a coded item either"?